### PR TITLE
fix: add admin setting for AI generated call recording summary

### LINKED ISSUE #13906: Add admin setting for AI generated call recording summary
> Frontend: toggle in admin settings 

 _Originally posted by @Antreesy in [#13429](https://github.com/nextcloud/spreed/issues/13429#issuecomment-2488438112)_

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix #13906 

### 🖼️ Screenshots / Screencasts

image

### 🏁 Checklist

- [ ] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [x] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

---

### 🚧 Tasks

- [ ] Should it change talk hash (RoomController.php)?

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included or not possible
- [ ] 📘 API documentation in `docs/` has been updated or is not required
- [ ] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/AdminSettings/RecordingServers.vue [hunk 1]
```diff
@@ -72,6 +72,24 @@
...
+			<template v-if="servers.length">
+				<h3>{{ t('spreed', 'Recording transcription') }}</h3>
+
+				<NcCheckboxRadioSwitch v-model="recordingTranscriptionEnabled"
+					type="switch"
+					:disabled="loading"
+					@update:modelValue="setRecordingTranscription">
+					{{ t('spreed', 'Automatically transcribe call recordings with a transcription provider') }}
+				</NcCheckboxRadioSwitch>
+
```
Comment (nickvergessen) [inline src/components/AdminSettings/RecordingServers.vue:85]:
```suggestion
```

We should hide this as the quality is still too bad at the moment

Comment (Antreesy) [reply src/components/AdminSettings/RecordingServers.vue:85]: I'd rather hide it completely until then

Comment (nickvergessen) [reply src/components/AdminSettings/RecordingServers.vue:85]: Then lets do that

**Comment by Antreesy** [comment]
Kept in template to have translation ready:

```html
	<!-- FIXME hidden until transcription quality is appropriate -->
	<NcCheckboxRadioSwitch v-if="false"
```

**Review by nickvergessen — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Pull request merged**